apps/storybook-react-native/jest.config.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/unambiguous */
```

Comment:
We can likely remove this file as we won't be writing jest tests for the storybook app